### ISSUE TITLE
Suggestion: Change function of the delay operator. 

### ISSUE THREAD

**Issue by Kyrobi** [kangarko/ChatControl#1629]

**Comment by Kyrobi** [comment]
Notice: Some of the best ideas and features we added were thanks to your suggestions. However, as an extremely small team we're focusing on stability and performance of our software first before adding new features. As such, we're currently very limited to taking in new features.

If you still believe your feature should be added, you're welcome to suggest it and answer those questions in your description: 

1. How it would benefit others? (Not just you, but 1,000s of others people using the same plugin.)
This will benefit server owners as they will have more fine-grain control over custom command rules.

2. What would happen if we won't implement this feature? How's your server being affected?
I will have to probably commission a custom plugin to handle cooldowns since the commands defined in the rule are not technically "commands." This means that something like Essential's command cooldown feature does not actually work

Currently, the "delay" rule in the command rules allow certain defined commands to be delayed. However, this delay is a global delay and not an individual delay.

For example, I have a command my through the command rule called /hug. It allows a user to hug another person. I've set a delay of 10 seconds on it to prevent spamming of the command. But right now, the delay is global. If I were to hug someone and the 10-second cooldown kicks in, no one else is able to use the command either.
